neuralprophet/configure.py

```diff
@@ -285,7 +287,7 @@ def __post_init__(self):
             else:
                 log.info("Trend reg lambda ignored due to no changepoints.")
                 self.trend_reg = 0
-                if self.trend_reg_threshold > 0:
+                if self.trend_reg_threshold and self.trend_reg_threshold > 0:
```

Comment:
what exactly is this code supposed to do? why is > 0 not sufficient?

Reply:
yes, it could also be `None` which would fail if we compare `None > 0`

Reply:
<img width="634" alt="Screenshot 2022-11-29 at 6 00 16 PM" src="https://user-images.githubusercontent.com/536225/204689152-46011418-1c09-453c-97a3-96fcec6ddbe1.png">